_posts/2017/05/2017-05-15-EmpoweringOthers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "What is the reward for empowering others?"
+date: 2017-05-14
+time: "20:00:00"
+category: ["Community","Teaching","Education","Workshops","Mentoring"]
+---
+
+What is the reward for empowering others? Following a few round-table discussions with Software Carpentry instructors, I gained some insight into what drives us to enthusiastically volunteer our time. I came to the conclusion that many of us share a common sense of accomplishment when we empower others. We enjoy empowering others through mentoring and education. Software Carpentry gives us a fountain-full of opportunities for empowering others through community developed lessons, community supported workshops, and world-wide community conversations. We start out volunteering a little when there i time, but we soon carve out more and more space in our schedule for Software Carpentry events and activities that empower us to empower others.  
+
+I can't help but wonder if part of this is driven by a lack of reward for empowering others in the traditional academic setting. Compare the motivation of learners in a required semester-long class introductory class to learners in a intense two-day workshop teaching best practices for domain-specific research. I prefer to teach the later class because I'm certain it enhances my learner's education and research. Furthermore, we all know that many academic professors are rewarded for research while paid for teaching (something I know but still don't quite comprehend). It is awesome that the Software Carpentry Instructor is valued for  contributions in teaching that enable the research that furthers the careers of our learners. 
+
+I wonder who else share a rewarding sense of accomplishment for empowering others. I'm also curious to know what other commonalities unite our global community? 
```

Comment:
1. "who else shares" (not "share")
2. remove "to know" from final sentence, and question mark from the end - you're saying you're curious, not asking a question.
3. Close with "If you have thoughts, we'd welcome comments." or something like that.